roles/foreman/tasks/main.yaml

```diff
@@ -143,7 +143,7 @@
 
 - name: Configure Foreman Proxy
   theforeman.foreman.smart_proxy:
-    name: "{{ ansible_fqdn }}"
+    name: "{{ ansible_fqdn }}-pulp"
     url: "https://{{ ansible_fqdn }}:9090"
```

Comment:
Pulp doesn't run on port 9090 so shouldn't you rename the other entry?

Reply:
Is there a point to this change before #81 then?

Reply:
mostly "splitting things up in smaller chunks", but I can add it to 81 if you prefer